mediation-old/src/main/java/com/wso2telco/dep/mediator/impl/wallet/WalletRefundHandler.java

```diff
@@ -74,9 +71,7 @@ public boolean handle(MessageContext context) throws Exception {
         String clientCorrelator = null;
         String requestResourceURL = executor.getResourceUrl();
 
-        FileReader fileReader = new FileReader();
-        String file = CarbonUtils.getCarbonConfigDirPath() + File.separator + FileNames.MEDIATOR_CONF_FILE.getFileName();
-		Map<String, String> mediatorConfMap = fileReader.readPropertyFile(file);
+        Map<String, String> mediatorConfMap = ConfigFileReader.getInstance().getMediatorConfigMap();
```

Comment:
**General Comment:** Noticed in some places the mediatorConfigMap is defined at the class level once and used subsequently. In other places, it is referred to directly from ConfigFileReader. Better if we can maintain a consistency on the usage for code clarity.

Reply:
This is not a code refactoring related to the internal logic. This is in the out of the scope change. Report a refactoring ticket to do this type of changes. Via EXTGW-362, just deligating the config file reading logic to the newly implemented framework. correct me if I am wrong.